sdk/iot/azure-iot-modelsrepository/samples/get_models_sample.py

```diff
@@ -0,0 +1,58 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+from azure.iot.modelsrepository import (
+    ModelsRepositoryClient,
+    DEPENDENCY_MODE_TRY_FROM_EXPANDED,
+    DEPENDENCY_MODE_ENABLED,
+)
+import pprint
+
+dtmi = "dtmi:com:example:TemperatureController;1"
+dtmi2 = "dtmi:com:example:TemperatureController;2"
+
+# By default the clients in this sample will use the Azure Device Models Repository endpoint
+# i.e. https://devicemodels.azure.com/
+# See client_configuration_sample.py for examples of alternate configurations
+
+
```

Comment:
Can we add a couple samples here which show users examples of error handling with respect to `get_models`?